joomla/joomla-cms
# [com_search] review config global default options

### Summary of Changes

This PR intends to normalize config global default options in com_search.

The method used was:
1. Review cs of the config.xml file - also add missing default values
2. Check the current params in installation joomla.sql mysql
3. Add missing defaults to install sql

### Testing Instructions

0. Code review
1. Apply patch
2. Delete configuration.php
3. Install Joomla (no sample)
4. Create a menu item of "Search" type and check all "Use Global" have the global value.

### Documentation Changes Required

None.

### Added/changed defaults in the xml

changed: none

added:
- opensearch_name, opensearch_description: *empty*
